### PR TITLE
Update WGPU deps to rev 08e8d406c175579da5ef18c1abf4d6c00e2a9726

### PR DESCRIPTION
**Pull request by aloucks**
I also moved the `path` chunks in `Cargo.toml` to a single `[patch]` section so that it's one block to commment/uncomment for local development.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -20,27 +20,14 @@ use std::{
...
-    },
+    binding_model::TextureComponentType,
     device::{
         BIND_BUFFER_ALIGNMENT,
     },
     instance::{
         AdapterInfo,
         DeviceType,
     },
-    resource::{
```
Comment (kvark) [inline src/lib.rs]: how come we are no longer re-exporting these?

Comment (aloucks) [reply src/lib.rs]: They're included above in `pub use wgt::*;`

**Comment by kvark** [comment]
Bors retry

**Comment by kvark** [comment]
Bors r+

**Comment by kvark** [comment]
bors retry

**Comment by aloucks** [comment]
retry agian?

**Comment by kvark** [comment]
bors retry

**Pull request merged**